package.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "@okteto/react-oauth2-login",
-  "version": "1.0.1",
+  "version": "1.0.2",
   "description": "A React Component for OAuth2 Login",
   "main": "dist/index.js",
   "types": "src/index.d.ts",
   "scripts": {
-    "build": "NODE_ENV=production webpack -p --config webpack.config.dist.js",
-    "start": "NODE_ENV=development webpack-dev-server --config webpack.config.js",
+    "build": "NODE_ENV=production NODE_OPTIONS=--openssl-legacy-provider webpack -p --config webpack.config.dist.js",
+    "start": "NODE_ENV=development NODE_OPTIONS=--openssl-legacy-provider webpack-dev-server --config webpack.config.js",
```

Comment:
Could you give some context about why we need this `--openssl-legacy-provider`?

Reply:
With Node 18, there was some code (perhaps on an older action version) that failed. 
Since actions were updated and strategy was removed, we can also remove this and take a look at it later.

Reply:
Great, thanks for clarifying!